platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/main/java/com/swirlds/platform/test/consensus/framework/validation/ConsensusRoundValidation.java

```diff
@@ -99,10 +99,7 @@ public static void validateIterableRounds(
      * @param e2 the second event
      */
     private static void assertConsensusEvents(final String description, final EventImpl e1, final EventImpl e2) {
-        final boolean equal = Objects.equals(e1.getHashedData(), e2.getHashedData())
-                && Objects.equals(
-                        e1.getBaseEvent().getConsensusData(), e2.getBaseEvent().getConsensusData())
-                && e1.isWitness() == e2.isWitness();
+        final boolean equal = Objects.equals(e1.getBaseEvent(), e2.getBaseEvent()) && e1.isWitness() == e2.isWitness();
```

Comment:
with this change `GossipEvent::getConsensusData` no longer has any usages

Reply:
for the time being, yes. but it will be used in upcoming PRs, this is a PBJ record that will be part of the block stream